x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning.tsx

```diff
@@ -7,67 +7,93 @@
+import { DOCS_PREFIX } from '../../routes';
 
-import { RELEVANCE_TUNING_TITLE } from './constants';
 import { RelevanceTuningForm } from './relevance_tuning_form';
-import { RelevanceTuningLogic } from './relevance_tuning_logic';
+import { RelevanceTuningLayout } from './relevance_tuning_layout';
+
+import { RelevanceTuningLogic } from '.';
 
 interface Props {
   engineBreadcrumb: string[];
 }
 
+const EmptyCallout: React.FC = () => {
+  return (
+    <EuiEmptyPrompt
+      title={
+        <h2>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.emptyErrorMessageTitle',
+            {
+              defaultMessage: 'Tuning requires schema fields',
+            }
+          )}
+        </h2>
+      }
+      body={i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.emptyErrorMessage',
+        {
+          defaultMessage: 'Index documents to tune relevance.',
+        }
+      )}
+      actions={
+        <EuiButton
+          size="s"
+          color="primary"
+          href={`${DOCS_PREFIX}/relevance-tuning-guide.html`}
+          fill
+        >
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.emptyButtonLabel',
+            {
+              defaultMessage: 'Read the relevance tuning guide',
+            }
+          )}
+        </EuiButton>
+      }
+    />
+  );
+};
+
 export const RelevanceTuning: React.FC<Props> = ({ engineBreadcrumb }) => {
+  const { dataLoading, engineHasSchemaFields, unsavedChanges } = useValues(RelevanceTuningLogic);
   const { initializeRelevanceTuning } = useActions(RelevanceTuningLogic);
 
   useEffect(() => {
     initializeRelevanceTuning();
   }, []);
 
-  return (
-    <>
-      <SetPageChrome trail={[...engineBreadcrumb, RELEVANCE_TUNING_TITLE]} />
-      <EuiPageHeader>
-        <EuiPageHeaderSection>
-          <EuiTitle size="l">
-            <h1>{RELEVANCE_TUNING_TITLE}</h1>
-          </EuiTitle>
-          <EuiText>
-            <EuiTextColor color="subdued">
-              {i18n.translate(
-                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.description',
-                {
-                  defaultMessage: 'Set field weights and boosts',
-                }
-              )}
-            </EuiTextColor>
-          </EuiText>
-        </EuiPageHeaderSection>
-      </EuiPageHeader>
-      <EuiSpacer />
-      <FlashMessages />
+  const body = () => {
+    if (dataLoading) {
+      return <Loading />;
+    }
+
+    if (!engineHasSchemaFields) {
+      return <EmptyCallout />;
+    }
+
+    return (
       <EuiFlexGroup>
         <EuiFlexItem>
           <RelevanceTuningForm />
         </EuiFlexItem>
         <EuiFlexItem />
       </EuiFlexGroup>
-    </>
+    );
+  };
+
+  return (
+    <RelevanceTuningLayout engineBreadcrumb={engineBreadcrumb}>
+      <UnsavedChangesPrompt hasUnsavedChanges={unsavedChanges} />
```

Comment:
[optional, not a blocker] I think you could probably bake `UnsavedChangesPrompt` into the `RelevanceTuningLayout`, unless you think that's too hidden or confusing. Up to you!